### PR TITLE
a8n: Temporarily disable the credentials campaign type

### PR DESCRIPTION
**Pull request by ryanslade**
This change is intended to only be included in a patch release to work
around https://github.com/sourcegraph/sourcegraph/issues/7914

### CODE REVIEW

**Comment by ryanslade** [comment]
> Add it to the changelog, though, to say that it's been temporarily disabled in this version to not cause incorrect results, but it's been fixed in the upcoming 3.13 version.

Are you sure? I don't see any of the other 3.12.1 changes or even a 3.12.1 section in the CHANGELOG

**Comment by mrnugget** [comment]
> Are you sure? I don't see any of the other 3.12.1 changes or even a 3.12.1 section in the CHANGELOG

Now I'm not sure anymore. I thought every release we publish includes a changelog that describes it. Maybe the changelog on `master` is enough then? Best sync with distribution on this.

File: enterprise/internal/a8n/resolvers/resolver.go [hunk 1]
```diff
@@ -487,6 +487,9 @@ func (r *Resolver) PreviewCampaignPlan(ctx context.Context, args graphqlbackend.
 	if typeName == "" {
 		return nil, errors.New("cannot create CampaignPlan without Type")
 	}
+	if typeName == "credentials" {
```
Comment (tsenart) [inline enterprise/internal/a8n/resolvers/resolver.go]: I thought there was a constant for this somewhere in the backend.

**Comment by slimsag** [comment]
I am going to cut 3.12.2, I assume you want this PR included so merging and will include.

Please update the CHANGELOG in `master` to include an entry for this change.

**Comment by slimsag** [comment]
Ah, unfortunately CI is failing. If you can get this merged when you're next online @ryanslade I'll include this

File: enterprise/internal/a8n/resolvers/resolver.go (hunk 1 above)
Comment (ryanslade) [reply enterprise/internal/a8n/resolvers/resolver.go]: Yes, but it's not exported and wanted to keep this change as minimal as possible

**Comment by ryanslade** [comment]
@slimsag I've fixed the CI issues

**Pull request merged**